compiler/dyno/lib/resolution/Resolver.h

```diff
@@ -279,6 +279,9 @@ struct Resolver {
   // resolve a special op call such as tuple unpack assign
   bool resolveSpecialOpCall(const uast::Call* call);
 
+  // Figure out the type of a type-level && operation.
```

Comment:
How about just

> Resolve a || or && operation.

It returns the type but it has the side effect of resolving nested calls (storing results in byPostorder).